README.md

```diff
@@ -45,153 +45,59 @@ Requires Python 3.7+ and *one of* [PyTorch](https://pytorch.org/)(>=1.9) or [Ten
 pip install finetuner
 ```
 
-## [Documentation](https://finetuner.jina.ai)
```

Comment:
Why remove link to documentation?

Reply:
i think everyone is searching docs on top right like this: 
![0019DEA6-61C5-4589-BB7F-09F241CEDAAA](https://user-images.githubusercontent.com/9794489/150828170-178d2b32-af99-410b-95b4-806a012ffd0c.png) @tadejsv